src/Client.php

```diff
@@ -321,4 +321,26 @@ protected function determineException(ClientException $exception): Exception
 
         return $exception;
     }
+
+
+    /**
+     * List shared links of this user..
+     *
+     * If no path is given, returns a list of all shared links for the current user.
```

Comment:
Try to condense this block of comments to the essence. One sentence should be enough.